x-pack/plugins/enterprise_search/public/applications/app_search/components/documents/search_experience/search_experience.test.tsx

```diff
@@ -47,12 +49,51 @@ describe('SearchExperience', () => {
     expect(wrapper.find(SearchProvider).length).toBe(1);
   });
 
+  describe('when there are no selected filter fields', () => {
+    let wrapper: ShallowWrapper;
+    beforeEach(() => {
+      setFieldsInLocalStorage({
+        filterFields: [],
+        sortFields: ['a', 'b'],
+      });
+      wrapper = shallow(<SearchExperience />);
+    });
+
+    it('shows a customize callout instead of a button if no fields are yet selected', () => {
+      expect(wrapper.find(CustomizationCallout).exists()).toBe(true);
+      expect(wrapper.find('[data-test-subj="customize"]').exists()).toBe(false);
+    });
+
+    it('will show the customization modal when clicked', () => {
+      expect(wrapper.find(CustomizationModal).exists()).toBe(false);
+      wrapper.find(CustomizationCallout).simulate('click');
+
+      expect(wrapper.find(CustomizationModal).exists()).toBe(true);
+    });
+  });
+
+  describe('when there are selected filter fields', () => {
+    let wrapper: ShallowWrapper;
+    beforeEach(() => {
+      setFieldsInLocalStorage({
+        filterFields: ['a', 'b'],
+        sortFields: ['a', 'b'],
+      });
+      wrapper = shallow(<SearchExperience />);
+    });
+
+    it('shows a customize button', () => {
+      expect(wrapper.find(CustomizationCallout).exists()).toBe(false);
+      expect(wrapper.find('[data-test-subj="customize"]').exists()).toBe(true);
+    });
```

Comment:
[Not a blocker since tests are already passing & at 100] Should we add a test for checking that filter checkboxes actually render as well? Something like `expect(wrapper.find(Facet)).toHaveLength(2)`?

Reply:
https://github.com/elastic/kibana/pull/88764/commits/433a7611a7226e6d401d8e26da6574aaadc00414